.github/workflows/e2e-nvidia-a10g-x4.yml

```
@@ -164,6 +164,18 @@ jobs:
         env:
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Post job results to Slack
```

```suggestion
      - name: Post job results to Slack
        if: ${{ always() }}
```

You want this step to run even when a prior step fails.